src/znc.cpp

```diff
@@ -338,6 +338,18 @@ bool CZNC::IsHostAllowed(const CString& sHostMask) const {
 	return false;
 }
 
+bool CZNC::IsBindHostAllowed(const CString& sBindHost) const {
```

Comment:
This is as useless as it was before this PR.
If sBindHost is empty, OS is free to choose *any* IP and connect from it. And user is always free to make bindhost empty regardless of this list.